### ISSUE TITLE
Oh,oh))      Irrelevant zeros are not automatically removed

### ISSUE THREAD

**Issue by free777cat** [shaise/FreeCAD_SheetMetal#413]
### Describe your issue

![Зняток екрана як 2024-12-04 11-05-28](https://github.com/user-attachments/assets/5f6ef232-c0a1-41cf-9555-501711c5b40b)

### FreeCAD version info + SheetMetal WB version

```shell
[code]
OS: Linux Mint 22 (MATE/mate)
Word size of FreeCAD: 64-bit
Version: 2024.1006.0.14555 (Git shallow) AppImage
Build type: Release
Branch: (HEAD detached at 20241003stable)
Hash: 8058dcf52c2661edabf36d8147d5b8a187378d2d
Python 3.11.9, Qt 5.15.13, Coin 4.0.1, Vtk 9.2.6, OCC 7.7.2
Locale: Ukrainian/Ukraine (uk_UA)
Installed mods: 

  * sheetmetal 0.6.0
```

**Comment by shaise** [comment]
I don't think its a sheetmetal issue. 
It does not happen for me in main and not in realthunder:
![image](https://github.com/user-attachments/assets/9e7a0f31-b558-42a7-ba49-d0198d6fc963)

![image](https://github.com/user-attachments/assets/66633709-9de4-4499-b71d-a897694a75dd)

**Comment by free777cat** [comment]
Does your program not work like this video?
https://youtu.be/_wfYyHwqoRs

**Comment by shaise** [comment]
Yes indeed but this is same for all other freecad parameters. not only sheetmetal. for example:
in part design:
![image](https://github.com/user-attachments/assets/5e01ce85-8ec1-431f-89fe-33c19c8095ec)

**Comment by free777cat** [comment]
This is not very critical.  But it works in the editor.
https://youtube.com/shorts/C7976aeWk_Y

**Comment by shaise** [comment]
Yes, you are correct. But as said, I use Freecad's Edit boxes, so I have no way to change their behavior.

**Comment by shaise** [comment]
Don't know...

**Comment by hasecilu** [comment]
Is your FreeCAD setting set corrrectly? I set 3 decimals and that's what I get

![image](https://github.com/user-attachments/assets/5227bf62-51a9-4d94-90cf-e0afb0d13a50)

**Comment by shaise** [comment]
@hasecilu , His issue is not with the number of zeros after the point but rather from the option to type zeros before the number
